src/igvfd/schemas/measurement_set.json

```diff
@@ -269,6 +269,22 @@
                 "appscript": "[\"ENSMUSG00000044487\", \"john-doe:gene01\"]",
                 "igvf_utils": "ENSMUSG00000044487, john-doe:gene01"
             }
+        },
+        "functional_assay_mechanisms": {
```

Comment:
Did we ask the lab to come up with this name? or did  we coin it? Just curious if we could use some other term for mechanisms. Maybe it is okay

Reply:
ah yes, as I replied in the first comment, they suggested the name.